test/docfx.Test/lib/GithubUtilityTest.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.Docs.Build
+{
+    public class GithubUtilityTest
```

Comment:
>Github [](start = 17, length = 6)

naming consistency, GitHub

Reply:
thanks!

---
In reply to: [224650596](https://github.com/dotnet/docfx/pull/3485#discussion_r224650596) [](ancestors = 224650596)